mantidproject/mantid
# Indirect - QENS Fitting - QENSFitSimultaneous algorithm

This PR adds a general algorithm for performing QENS simulataneous fitting - QENSFitSimultaneous. This algorithm performs all common functionality for QENS simultaneous fitting algorithms. Limited QENS simultaneous fitting currently only exists for I(Q,t) data in the form of IqtFitMultiple (only for a single dataset and across a continuous spectra range).

This algorithm is a wrapper for the underlying 'Fit' algorithm and is able to accept multiple datasets and discontinuous spectra ranges as input.

**To test:**
1. Ensure the unit tests pass.
2. Run the provided `qens_simultaneous_fit.py` script, ensuring the output format mirrors that of QENSFitSequential (see #22304).


**Test Files**
[QENSFitTestFiles.zip](https://github.com/mantidproject/mantid/files/2010145/QENSFitTestFiles.zip)



Fixes #22056. 

**Release Notes** 
Release notes have been updated


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
